cmd/gentkn/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+
+	"time"
+
+	"github.com/ardanlabs/service/internal/platform/auth"
+	jwt "github.com/dgrijalva/jwt-go"
+	"github.com/google/uuid"
+	"github.com/kelseyhightower/envconfig"
+)
+
+func main() {
+
+	// Define and process configuration for entire tool.
+	var cfg struct {
+		KeyID string `envconfig:"KEY_ID" default:"default"`
+
+		Claims struct {
+			Audience string   `envconfig:"AUDIENCE"`
+			Subject  string   `envconfig:"SUBJECT"`
+			Roles    []string `envconfig:"ROLES"`
+		} `envconfig:"CLAIMS"`
+	}
+	if err := envconfig.Process("GENTKN", &cfg); err != nil {
+		log.Fatalf("main : Parsing Config : %v", err)
+	}
+
+	/* TODO: Fix non-flag processing in flag pkg.
+	if err := flag.Process(&cfg); err != nil {
+		if err != flag.ErrHelp {
+			log.Fatalf("main : Parsing Command Line : %v", err)
+		}
+		return // We displayed help.
+	}
+	*/
+
+	// Source the private key.
+	var keyRdr io.Reader
+	if len(os.Args) < 2 {
+		log.Fatal("main : Missing Required Param <key-file>")
+	}
+	switch file := os.Args[1]; file {
```

Comment:
I think we need a line feed here and it's probably a good place for that variable.